en/_posts/2016-02-28-using-timelion-with-laas.md

```diff
@@ -0,0 +1,113 @@
+
+All the informations about this wonderful plugin can be found here :
+
+
+[https://www.elastic.co/blog/timelion-timeline](https://www.elastic.co/blog/timelion-timeline).
+
+
+We strongly believe that you will love this tool if you need to analyze your datas on PaaS Logs. 
+
+In order to be able to use Timelion, you have to your Kibana access configured. If you don't have it already, you can fly [here](/kb/en/logs/using-kibana-with-laas.html). 
+If you're ready, let's get started !
+
+#1 Installing Timelion on Kibana 
+
+The [documentation](https://github.com/elastic/timelion#installation) will help you through the installation process, but here is how do to do it actually : 
+
+ - stop your kibana
+ - issue the following from your kibana installation folder `./bin/kibana plugin -i elastic/timelion`
+ - restart Kibana. you should see a new Apps Icon Appears, if you click on it you will see Timelion. 
+
+
+![timelion](/kb/images/2016-02-28-using-timelion-with-laas.md/timelion.png)
+
+
+#2 Configuring Timelion for PaaS Logs
+
+
+By default, timelion use the index \_all to query on your indexes and the field "@timestamp" as the date field. In order to allow it to use the correct settings, do the following : 
+ - In your Kibana Installation, Edit the file located at "installedPlugins/timelion/timelion.json". 
+ - Replace timefield with: "timestamp", replace default\_index with the alias you want to use. 
+ - Restart Kibana
+Note that if you have several aliases, you can of course query them in Timelion using the following syntax .es(index="my\_other\_alias").
+
+
+#3 Where do I go from here ?
+
+Timelion has a built in help and tutorial to help get you started. To access it, use the tiny "?" icon at the far-right of the interface (just under the main search bar). If you don't see it, use the toggle menu button at the right of the search bar to display it. 
+
+the Elasticsearch commands start by ".es", you can change the resolution of the chart by using the drop down menu at the right of the search bar.  
+
+ - To display all your data at the selected timerange (top right), use:
+
+```
+.es(*)
+```
+
+ - To display only the data that have a certain field use
+
+```
+.es(field:value)
+```
+
+ - To display the average on a numeric value present in your logs use : 
+
+```
+.es(metric='avg:my_field_num')
+```
+
+ - You can display only the variation (derivative) on this value by using the following formula : 
+
+```
+.es(metric='avg:my_field_num').derivative()
+```
+
+ - To display different data on different yaxis, use the yaxis() functions.
+
+```
+.es(metric='avg:my_field_num').derivative(), es(*).yaxis(2)
+```
+
+In the following screenshot, you can easily see if there is a correlation or not between different metrics we have in your softwares (here we tried to find one in HA Proxy between the variation of the bytes\_read and the duration of requests). 
```

Comment:
"if there is a correlation or not" you can skip the "or not" this will help the reader by making the sentence easier